modules/store/src/state.ts

```diff
@@ -50,6 +55,8 @@ export class State<T> extends BehaviorSubject<any> implements OnDestroy {
       this.next(state);
       scannedActions.next(action as Action);
     });
+
+    this.state = toSignal(this, { manualCleanup: true });
```

Comment:
we forgot to add `requireSync: true` when we moved from custom to Angular's `toSignal` function

```suggestion
    this.state = toSignal(this, { manualCleanup: true, requireSync: true });
```

Reply:
Why is this? 👀

Reply:
> The initial (custom) version of the toSignal function that we used had the same behavior as we now have 

Woops, I should've known that 😅
Thanks Marko!